app/models/miq_server/environment_management.rb

```diff
@@ -29,7 +29,7 @@ def minimal_env_options
         options.shift # remove the "minimal" from the front of the array
 
         # Special case where Netbeans is handling the UI worker for debugging
-        options.collect { |o| o.downcase == "netbeans" ? %w(schedule reporting noui) : o }.flatten
+        options.collect { |o| o.downcase == "netbeans" ? %w(schedule reporting no_ui) : o }.flatten
```

Comment:
Totally; your point is equally valid :bow: 

Removing solves both; I'm for removing this entirely. We could add to the minimal mode docs on the site that you should run no_ui if you use a debugger such as Netbeans or RubyMine. @dclarizio @chessbyte ?